src/lib/shell.html

```diff
@@ -33,34 +33,37 @@
             </div>
         </div>
     </div>
-    <div class="bp" tabindex="0">
-        <div class="bp-loading-wrapper">
-            <div class="bp-loading">
-                <div class="bp-icon bp-icon-file"></div>
-                <div class="bp-crawler-wrapper">
-                    <div class="bp-crawler">
-                        <div></div>
-                        <div></div>
-                        <div></div>
+    <div class="bp-body">
+        <div class="bp-thumbnails-container bp-is-hidden"></div>
```

Comment:
Does this need to be in the shell? Can it be appended to the dom dynamically?

Reply:
Our pattern for most big pieces of UI has been to add them hidden and toggle as needed. Not saying it's a good thing, but it's consistent.

Reply:
Talked to Daniel about this. He's right, we dynamically generate viewer specific things like the print popup.